spec/lib/auth0/api/v2/client_grants_spec.rb

```diff
@@ -22,10 +23,11 @@
     it 'is expected to send get /api/v2/client-grants/ with pagination' do
       expect(@instance).to receive(:get).with(
         '/api/v2/client-grants',
+        client_id: '1',
         page: 1,
         per_page: 2
       )
-      expect { @instance.client_grants(page: 1, per_page: 2) }.not_to raise_error
```

Comment:
keep this test, and add a new one for each introduced property. This way you fix the "Codecov: Changes" failing status check.

Reply:
Sure. Thank you!